tests/diff.rs

```diff
@@ -101,8 +102,10 @@ mod gas {
 				run_diff_test("gas", concat!(stringify!($name), ".wat"), |input| {
 					let rules = instrument::gas_metering::ConstantCostRules::default();
 
-					let module =
+					let module: Module =
 						elements::deserialize_buffer(input).expect("Failed to deserialize");
+					let module: Module = module.parse_names().expect("Failed to parse names");
```

Comment:
The second one shouldn't be necessary?

```suggestion
					let module = module.parse_names().expect("Failed to parse names");
```